src/org/zalando/stups/pierone/api_v1.clj

```diff
@@ -236,7 +236,10 @@
                 tar-stream (TarArchiveInputStream. (GzipCompressorInputStream. fis))]
       (loop []
         (when-let [entry (.getNextTarEntry tar-stream)]
-          (if (= (.getName entry) "scm-source.json")
+          (if (or (= (.getName entry)
+                     "./scm-source.json")
+                  (= (.getName entry)
```

Comment:
how about
(if-not (nil? (re-matches #"(\.\/)?scm-source\.json" (.getName entry))) (extract...) (recur))